bin/sra_runinfo_to_ftp.py

```diff
@@ -6,7 +6,6 @@
 import argparse
 import collections
```

Comment:
same for this script...

Reply:
Fixed https://github.com/nf-core/rnaseq/pull/660/commits/57f3a4760ebd78fd4252458f3f8088d6bef5c2c6